packages/frontend/core/src/providers/modal-provider.tsx

```diff
@@ -183,6 +184,44 @@ export const SignOutConfirmModal = () => {
   );
 };
 
+export const AIReloadConfirmModal = () => {
+  const editorSettingService = useService(EditorSettingService);
+  const enableAI = useLiveData(
+    editorSettingService.editorSetting.settings$
+  ).enableAI;
+  const [aiState, setAiState] = useState(enableAI);
+  const [open, setOpen] = useState(false);
+
+  useEffect(() => {
+    if (enableAI !== aiState) {
```

Comment:
`setOpen(enableAI !== aiState)`, if close and open it immediately, the confirm modal in other tabs could be closed automatically